openandroidinstaller/assets/configs/miatoll.yaml

```diff
@@ -0,0 +1,64 @@
+metadata:
+  maintainer: A non (anon)
+  device_name: Xiaomi Redmi Note 9S / 9 Pro / 9 Pro Max / 10 Lite / Poco M2 pro
+  is_ab_device: false
+  device_code: miatoll
+  supported_recovery:
+    - orangefox
+    - twrp
+  supported_device_codes:
+    - miatoll
+    - gram
+    - curtana
+    - excalibur
+    - joyeuse
+  notes: >
```

Comment:
Shouldn't this be changed to `brand: xiaomi` now?

Reply:
Yes, done for all devices.